app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,13 @@
+<% provide(:title, "About | ") %>
+<h1>About</h1>
+<p>
+  The <% link_to "<em>Ruby on Rails
+  Tutorial</em>", "http://www.railstutorial.org/" %>, part of the
```

Comment:
You should using gem config to manage link address

https://github.com/rubyconfig/config

Reply:
Done